components/resourceDetails/resourceSharing/agentAccess/agentAccessOptionsMenu/removeButton/index.test.jsx

```diff
@@ -22,15 +22,20 @@
 import React from "react";
 import { waitFor } from "@testing-library/dom";
 import userEvent from "@testing-library/user-event";
+// eslint-disable-next-line camelcase
+import { acp_v2 } from "@inrupt/solid-client";
 import { renderWithTheme } from "../../../../../../__testUtils/withTheme";
 import mockConfirmationDialogContextProvider from "../../../../../../__testUtils/mockConfirmationDialogContextProvider";
 import mockAccessControl from "../../../../../../__testUtils/mockAccessControl";
 import RemoveButton, {
   handleConfirmation,
   handleRemovePermissions,
 } from "./index";
+import { PUBLIC_AGENT_PREDICATE } from "../../../../../../src/models/contact/public";
+import { AUTHENTICATED_AGENT_PREDICATE } from "../../../../../../src/models/contact/authenticated";
```

Comment:
Note for later: let's set up webpack to be able to resolve `src/*` instead of a billion `../`s